app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,16 @@
+class LikesController < ApplicationController
+  def create
+    @post = Post.find(params[:post_id])
+    current_user = User.find(params[:user_id])
```

Comment:
**OPTIONAL** - You did well so far, but please use the `current_user` method in your application_controller, please fix this 👍 👍

Reply:
done and thank you very much